src/libraries/System.Net.Security/src/System/Net/NegotiateAuthenticationPal.Unix.cs

```diff
@@ -19,7 +19,7 @@ namespace System.Net
 {
     internal partial class NegotiateAuthenticationPal
     {
-        private static bool UseManagedNtlm { get; } =
+        public static bool UseManagedNtlm { get; } =
```

Comment:
I don't think this should be flipped to `public`.

Are you adding an ILLink.Substitutions.xml entry for this new feature switch? Here's an example:

https://github.com/dotnet/runtime/blob/0c4329bcb6f287d24d3b117af7199956d9d590ac/src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.xml#L1-L15

Reply:
I can revert it but it doesn't really matter since it's in non-public class. I had trouble getting `private` property to work correctly but it seems to have been an issue with my test setup since I cannot reproduce it anymore.